core-kotlin-modules/core-kotlin-lang-oop/src/main/kotlin/com/baeldung/companionobject/CompanionObject.kt

```diff
@@ -0,0 +1,62 @@
+package com.baeldung.companionobject
+
+class CompanionObject {
+    class ClassName {
+        fun anotherFun() {
+            println(propertyName)
+        }
+
+        companion object {
+            const val propertyName: String = "Something..."
```

Comment:
Let's annotate this property with `@JvmStatic` like the document:

```suggestion
        companion object {
            // companion object members can be marked @JvmStatic for better java inter-op
            @JvmStatic
            const val propertyName: String = "Something..."
```

Reply:
ok, now propertyName is equipped with @JvmStatic.